test.py

```diff
@@ -31,17 +33,41 @@ def checkSelenium():
     except ImportError:
         prRed("❌ Selenium not present. Install Selenium: https://pypi.org/project/selenium/")
 
-def checkFirefox():
+# def checkFirefox():
```

Comment:
No comments like this in code, why has it been commented?

P.S. Thank you for your contributions @Yembot31013 !